pkg/apis/core/v1alpha1/types.go

```diff
@@ -250,6 +250,45 @@ const (
 // DataVolumeCloneSourceSubresource is the subresource checked for permission to clone
 const DataVolumeCloneSourceSubresource = "source"
 
+//CDIStorageClassCapabilities provides a CDI specific extension for StorageClasses
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=cdistorageclass;cdistorageclasses,scope=Cluster
+type CDIStorageClass struct {
```

Comment:
I would name it CDIStorageClassCapabilities. The current name implies it is some kind of special storage class, while in fact it is defining properties of actual storage classes.

Reply:
after discussion settled on the name StorageProfile